app/src/main/AndroidManifest.xml

```diff
@@ -50,6 +50,25 @@
 
         <receiver android:name=".Receivers.NotificationAlarmReceiver" />
 
+        <receiver
+            android:name=".widget.BookmarkWidgetProvider"
+            android:enabled="true"
+            android:label="Bookmarks">
+            <intent-filter>
+                <action android:name="android.appwidget.action.APPWIDGET_UPDATE" />
+                <action android:name="org.fossasia.openevent.ACTION_DATA_UPDATED" />
```

Comment:
Probably should use the `${applicationId}` placeholder here since it is set dynamically in [app/build.gradle](app/build.gradle). Same goes for wherever `org.fossasia` has been used.